src/main/scala/fi/oph/koski/koskiuser/LogoutServlet.scala

```diff
@@ -20,51 +20,28 @@ class LogoutServlet(implicit val application: KoskiApplication) extends Virkaili
     removeUserCookie
 
     if (virkailija) {
-      redirectToLogout
+      redirectToVirkailijaLogout
     } else {
       params.get("target") match {
-        case Some(target) => luvanluovutusLogout(target)
-        case None => kansalaisLogout
+        case Some(target) if target != "/" => {
+          redirectToOppijaLogout(target)
+        }
+        case _ => redirectToOppijaLogout(koskiRoot)
       }
     }
   }
 
-  private def kansalaisLogout = {
-    val shibbolethLogoutUrl = LogoutServerConfiguration.shibbolethLogoutUrl(application, langFromDomain)
-    if (shibbolethLogoutUrl.isEmpty) {
-      redirectToFrontpage
-    } else {
-      redirect(shibbolethLogoutUrl)
-    }
-  }
-
-  private def luvanluovutusLogout(target: String) = {
-    val redirectToTargetUrl = "/koski/user/redirect?target=" + encode(target)
-    val shibbolethLogoutUrl = LogoutServerConfiguration.configurableShibbolethLogoutUrl(application, langFromDomain)
-    val url = if (shibbolethLogoutUrl.isEmpty) {
-      redirectToTargetUrl
-    } else {
-      shibbolethLogoutUrl + encode(redirectToTargetUrl)
-    }
-    redirect(url)
-  }
-
   private def encode(param: String) = URLEncoder.encode(param, "UTF-8")
 }
 
 object LogoutServerConfiguration {
```

Comment:
Nähdäkseni tätä LogooutServerConfiguration luokkaa ei enää käytetä muualla kuin testeissä. Poista kokonaan, jos näin?